prelab_setup.Rmd

```diff
@@ -0,0 +1,6 @@
+# Pre-lab
```

Comment:
Consider formally adding section titles e.g. Purpose, Learning Objectives, Introduction, Activity 1 -- Set up AnVIL Account, Activity 2 -- Access Materials

Reply:
made an issue for this #22